samples/authenticator/src/main/java/org/forgerock/authenticator/sample/camera/CameraScanActivity.java

```diff
@@ -44,40 +50,45 @@
 
     private static final String TAG = CameraScanActivity.class.getSimpleName();
 
+    private PreviewView defaultView;
```

Comment:
Looks good to me, general note I would prefer not to use generic property names like "defaultView" but that's a minor. Check the commented out code above please as well.

Reply:
I agree. But the "defaultView" in this case means this will be default view to be used in case Google Play Services is available. Otherwise it will use an alternative view.